cpp/src/parquet/schema_test.cc

```diff
@@ -171,17 +167,16 @@ TEST_F(TestPrimitiveNode, Attrs) {
 }
 
 TEST_F(TestPrimitiveNode, FromParquet) {
-  SchemaElement elt =
-      NewPrimitive(name_, FieldRepetitionType::OPTIONAL, Type::INT32, field_id_);
+  SchemaElement elt = NewPrimitive(name_, FieldRepetitionType::OPTIONAL, Type::INT32);
```

Comment:
I'm not sure I understand this change in the tests. The user should still be able to pass an explicit `field_id` when creating a schema node, no?

Reply:
No, the user does not need this capability.  See comment below.

Reply:
Well, let's say I want to create a `Node` with a given field id as was done in this test. Would you I do that?